### PR TITLE
Avatar components

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

**Review by Namrata-J — APPROVED** [review]

File: components.css [hunk 1]
```diff
@@ -61,4 +61,144 @@
...
+    margin: 1rem;
+    height: 4rem;
+    border-radius: 1rem;
+}
+
+.square-x-small-avatar{
+    box-shadow: 5px 5px 15px var(--black);
+    margin: 1rem;
+    height: 3rem;
+    border-radius: 1rem;
```
Comment (Namrata-J) [inline components.css:119]: you can use utility classes for border-radius, as you are applying same proprty again and again in every declaration block.

**Review by bhanuprakashj — COMMENTED** [review]
code looks good to me !! 
Suggestion: try and use common classes instead of repeating same css across all avatars, increases code maintainability

File: components.css (hunk 1 above)
Comment (ganesh-kanchi) [reply components.css:119]: ok will follow that, thanks for the review.

**Comment by ganesh-kanchi** [comment]
> Reference

Thought about it halfway through, will keep it in mind during the next component. thanks for the review though.

**Pull request merged**